editor/src/components/canvas/controls/tailwindclassnames.ts

```diff
@@ -0,0 +1,12 @@
+`
+
+export const TailWindList = classNamesAsString
+  .split(',')
+  .filter((name) => {
+    return name.startsWith('.')
+  })
+  .map((name) => {
+    return name.slice(1)
+  })
```

Comment:
Please can you manually run this and replace this file so that we just have `export const TailwindClasses = [ ...`

Reply:
I'd like to second this because really long lines make for sad times when searching the project.

Reply:
plus I can imagine this _could_ have a measurable impact on editor startup time!

Reply:
fixed